cypress/integration/ResultsPagerSpanish.feature

```diff
@@ -9,10 +9,8 @@ Feature: As a user, I would like to be able to page through search results from
     And both pagers display numbers 1 and 2, followed by "...", the last page number and the option to click "Siguiente >" and "Ir a la Página" for screen readers
     And user clicks "Siguiente" and displays 1, 2, and 3 followed by "...", and 2 highlighted as the page they are on and "Ir a la Página" for screen readers and 560 and "Siguiente >"
     And the last page number to be visible
-    And the option for "< Anterior" appears before the page numbers
```

Comment:
same as above

Reply:
Previous will not show until you get to 3..